zoo/repos/entities_yaml.py

```diff
@@ -0,0 +1,151 @@
+from pathlib import Path
+from typing import List, Union
+
+import structlog
+import yaml
+from jsonschema import ValidationError
+from jsonschema import validate as schema_validate
+from yaml import FullLoader, load_all, safe_dump_all
+
+from zoo.repos.models import Repository
+from zoo.repos.utils import delete_none
+
+log = structlog.get_logger()
+
+
+SCHEMAS = {
+    "base_component": str(Path(__file__).parents[1])
+    + "/entities/yaml_definitions/component_base.yaml",
+    "service": str(Path(__file__).parents[1])
+    + "/entities/yaml_definitions/component_service.yaml",
+    "library": str(Path(__file__).parents[1])
+    + "/entities/yaml_definitions/component_library.yaml",
+}
+
+
+def get_schema(entity):
+    if entity["kind"] == "component":
+        if entity["spec"]["type"].lower() == "service":
```

Comment:
Can you eliminate these 2 ifs with just a single one?
In pseudo-code something like:
```
if contains(SCHEMAS, entity["spec"]["type"].lower() then
    return SCHEMAS[entity["spec"]["type"].lower()]
endif
```

Reply:
technically yes, logically it was ment to be separate `kind` and `type`. Now we have only `component` kind but could have more in future...

Reply:
I was referencing the 2 type ifs being replaced. You leave the kind if